### PR TITLE
Update configure-exchange-server-for-hybrid-modern-authentication.md

### PR DESCRIPTION
**Pull request by msbemba**
added IP ranges to whitelist

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Enterprise/configure-exchange-server-for-hybrid-modern-authentication.md [hunk 1]
```diff
@@ -149,6 +149,12 @@ Once you enable HMA, a client's next login will use the new auth flow. Note that
 You should also hold down the CTRL key at the same time you right click the icon for the Outlook client (also in the Windows Notifications tray) and click 'Connection Status'. Look for the client's SMTP address against an 'Authn' type of 'Bearer\*', which represents the bearer token used in OAuth.
   
  **Note** Need to configure Skype for Business with HMA? You'll need two articles: One that lists [supported topologies](https://docs.microsoft.com/skypeforbusiness/plan-your-deployment/modern-authentication/topologies-supported), and one that shows you [how to do the configuration](configure-skype-for-business-for-hybrid-modern-authentication.md).
+ 
+## Using hybrid Modern Authentication with Outlook for iOS and Android
+
+Customer on premise Customer when using Exchange server on TCP 443 should whitelist the below IP ranges
```
Comment (JohanFreelancer9) [inline Enterprise/configure-exchange-server-for-hybrid-modern-authentication.md]:
```suggestion
If you are an on-premises customer using Exchange server on TCP 443, please whitelist the following IP ranges:
```

File: Enterprise/configure-exchange-server-for-hybrid-modern-authentication.md [hunk 2]
```diff
@@ -149,6 +149,12 @@ Once you enable HMA, a client's next login will use the new auth flow. Note that
...
+
```
Comment (JohanFreelancer9) [inline Enterprise/configure-exchange-server-for-hybrid-modern-authentication.md]:
```suggestion
```

**Comment by mypil** [comment]
@msbemba  - please merge the commit on the copy/edit review done by JohanFreelancer9  or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by mypil** [comment]
@kelleyvice-msft - PR has been copy edited and is ready for final review. This is based on issue #596

Thank you.

cc: @kenwith

**Pull request merged**